src/domain/usecases/UpdateEventDataValueUseCase.ts

```diff
@@ -0,0 +1,104 @@
+import _ from "lodash";
+import { createObjectCsvWriter } from "csv-writer";
```

Comment:
Code in the domain cannot use external dependencies like csv-writer. Proposal: `EventExportSpreadsheetRepository implements EventExportRepository`